docs/tutorials/getting-started-hix.md

```diff
@@ -1,227 +1,17 @@
 # Getting started with Hix
 
-The `hix` tools are wrappers for the various `nix` tools that
-use `haskell.nix` without the need to add any `.nix` files.
+Hix is a command line tool that provides an easy way to add haskell.nix
+support to existing haskell projects.
 
-This is useful for:
+The `hix init` command adds a `flake.nix` and `nix/hix.nix` file.
+After that the project can be used with regular `nix` tools.
 
-* A quick way to try out haskell.nix for new users.
+For instance to run `cabal build` on the `hello` package from hackage:
 
-* Using haskell.nix to work on projects that do not have
-  `.nix` files.
-
-* Testing to see if `haskell.nix` can build a project.
-
-* Making `flake` and `non flake` configurations to check `haskell.nix`
-  treats them the same.
-
-## Installing Nix
```

Comment:
Maybe we can replace this deleted section by a quick link to the one that helps to install Nix and set up the IOG binary cache?